### PR TITLE
Add umeehub and umeeosmo paths

### PR DESCRIPTION
**Pull request by facundomedica**

### CODE REVIEW

File: interchain/umee-1/umeeosmo.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      "client-id": "07-tendermint-6",
+      "connection-id": "connection-0",
+      "channel-id": "channel-0",
+      "port-id": "transfer",
+      "order": "unordered",
+      "version": "ics20-1"
+    },
+    "dst": {
+      "chain-id": "cosmoshub-4",
+      "client-id": "07-tendermint-1805",
```
Comment (boojamya) [inline interchain/umee-1/umeeosmo.json:13]: Everything looks good except this light client. 

Can you verify `07-tendermint-1805`

**Review by boojamya — CHANGES_REQUESTED** [review]

**Comment by boojamya** [comment]
FYI, we are slowly starting to migrate all this path meta-data over to the chain registry. 

Akash Example: https://github.com/cosmos/chain-registry/blob/master/akash/paths.json

Keep an eye out for this in the future.

File: interchain/umee-1/umeeosmo.json (hunk 1 above)
Comment (boojamya) [reply interchain/umee-1/umeeosmo.json:13]: Actually, client seems correct. Should just need to change chain-id to `osmosis-1`

Comment (facundomedica) [reply interchain/umee-1/umeeosmo.json:13]: TY! 🙏

**Review by boojamya — APPROVED** [review]

**Pull request merged**